node/src/components/consensus/highway_core/highway/vertex.rs

```diff
@@ -148,6 +149,7 @@ impl<C: Context> Debug for WireVote<C> {
             .field("timestamp", &self.timestamp.millis())
             .field("panorama", self.panorama.as_ref())
             .field("round_exp", &self.round_exp)
+            .field("endorsed_votes", &self.endorsed)
```

Comment:
Let's keep the `Debug` impl in line with the code as much as possible:
```suggestion
            .field("endorsed", &self.endorsed)
```